src/test/scala/io/iohk/ethereum/vm/PrecompiledContractsSpec.scala

```diff
@@ -46,6 +47,30 @@ class PrecompiledContractsSpec extends FunSuite with Matchers with PropertyCheck
     gasUsedFailedRecover shouldEqual 3000
   }
 
+  test("ECDSARECOVER_vPrefixedf0") {
```

Comment:
What does it mean?

Reply:
Hmm I have just copied it from `blockchainsuite`, I will rename it as someone without context may not decipher it